UnitedIncome/components
# ItemizationBox CSS

### Description
> Please provide a brief description of your changes below.

Just some minor changes to the ItemizationBox CSS as it wasn't fit for purpose.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Should look fine in Storybook.

### Comments
> Any additional comments that will aid with the review process.

N/A
